torch/_dynamo/symbolic_convert.py

```diff
@@ -1341,6 +1347,34 @@ def MAP_ADD(self, inst):
             ),
         )
 
+    def SET_ADD(self, inst):
+        v = self.pop()
+        assert inst.argval > 0
+        obj = self.stack[-inst.arg]
+        assert isinstance(obj, SetVariable)
+        assert obj.mutable_local
+        # only copy if the new obj contains other mutables
+        new_rec_contains = obj.recursively_contains
+        if v.recursively_contains or v.mutable_local:
+            new_rec_contains = obj.recursively_contains.union(v.recursively_contains)
+
+            if v.mutable_local:
+                new_rec_contains.add(v.mutable_local)
+
+        nitems = set(obj.items)
+        nitems.add(v)
+        obj.items = list(nitems)
+        self.replace_all(
+            obj,
+            SetVariable(
+                obj.items,
+                mutable_local=obj.mutable_local,
+                recursively_contains=new_rec_contains,
+                regen_guards=False,
+                **VariableTracker.propagate([obj, v]),
```

Comment:
remind me what this does?  combines the guards of obj and v, or something?

Reply:
yea